### PR TITLE
Initial documentation (SOFTWARE-3199)

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

**Comment by matyasselmeci** [comment]
This isn't done because I need a template for a VO YAML file and we're still debating that in #1. I put it up here anyway for review.

**Comment by matyasselmeci** [comment]
OK, VO template added. Ready for review.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+The data is organized as files in [YAML format](https://en.wikipedia.org/wiki/YAML)
+as follows:
+
+-   `projects/` contains information about research projects that run jobs on OSG.
+    Each project has its own file, named `<PROJECT>.yaml`
+
+-   `virtual-organizations/` contains information about Virtual Organizations
+    (VOs).
+    Each VO has its own file, named `<VO>.yaml`.  In addition, there is a file
+    containing information about "reporting groups" called
```
Comment (brianhlin) [inline README.md]: We should try and flesh out some sort of definition for reporting groups

File: README.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    `REPORTING_GROUPS.yaml`.
+
+-   `topology/` contains information about the topology of the resources that
+    OSG sites provide.  Resources are collected into "resource groups."  Each
+    resource group has its own file, which are further organized by facility,
+    and then by site.  Resource group files are named
+    `<FACILITY>/<SITE>/<RESOURCEGROUP>.yaml`
```
Comment (brianhlin) [inline README.md]: Maybe use UW HEP or UNL as an example here?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+    Each project has its own file, named `<PROJECT>.yaml`
+
+-   `virtual-organizations/` contains information about Virtual Organizations
+    (VOs).
+    Each VO has its own file, named `<VO>.yaml`.  In addition, there is a file
+    containing information about "reporting groups" called
+    `REPORTING_GROUPS.yaml`.
+
+-   `topology/` contains information about the topology of the resources that
+    OSG sites provide.  Resources are collected into "resource groups."  Each
```
Comment (brianhlin) [inline README.md]: We should clarify what "resources" mean: `e.g. HTCondor-CE, GridFTP, XRootD, etc`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+`myosg.grid.iu.edu`.
+
+
+Updating the Data or Creating New Data
+--------------------------------------
+
+To update the data for your site, project, or VO, please either edit the
+matching YAML file and submit a GitHub pull request.  Or, if you are not
+comfortable directly making those changes, send email to **TODO**
+with the changes you want made.
```
Comment (brianhlin) [inline README.md]: We can use help@opensciencegrid.org

File: README.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+    `virtual-organizations/<VO NAME>.yaml`
+-   `template-resourcegroup.yaml` for new resource groups;
+    should be put in `<FACILITY>/<SITE>/<RESOURCE GROUP NAME>.yaml`.
+
+**Note**: File and directory names _must_ match the name of your project, VO,
+facility, site, or resource group, as appropriate.  This includes case and
+spaces.
+
+
+The Scripts
```
Comment (brianhlin) [inline README.md]: Everything from here down should go into a separate `README.md` in the `src/` directory

File: README.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+To update the data for your site, project, or VO, please either edit the
+matching YAML file and submit a GitHub pull request.  Or, if you are not
+comfortable directly making those changes, send email to **TODO**
+with the changes you want made.
+
+To create a new resource group, project, or VO, please use one of the template
+files to create the file in the appropriate directory, and fill out the
+information.  The comments in the template files should explain the structure
+and the meaning of the data.  If anything is confusing or you do not feel
+comfortable creating the new file yourself, send email to **TODO** with details
```
Comment (brianhlin) [inline README.md]: Use help@opensciencegrid.org, too

**Comment by matyasselmeci** [comment]
By "header to each yaml file," you mean the ones we convert?

File: README.md (hunk 1 above)
Comment (matyasselmeci) [reply README.md]: I hope someone can tell me the definition of a reporting group.

Comment (brianhlin) [reply README.md]: Marina and Rob are still around, one of them may know the anwer

**Comment by brianhlin** [comment]
Yup, the ones that we're generating off of the OIM DB. The alternative was creating a README.md in the project, VO, and site dir but that seems like a giant pain...

**Pull request merged**